index.html

```diff
@@ -4372,6 +4372,154 @@ <h2>Revision History</h2>
+  {&quot;verificationMethod&quot;: pubCEO, &quot;proofValue&quot;: signature(privCEO, document)}]
+}
+      </code></pre>
+      <p>
+We can introduce a &quot;dependency relation&quot; between <em>proofs</em> in a
+<em>proof set</em> by assigning the proof that another proof relies on, i.e.,
+the <em>dependent proof</em>, an <em>id</em> and the <em>relying proof</em> will
+have a list (or single item) denoting <em>previousProofs</em>. Such
+<em>dependency chains</em> can have arbitrary depth. The <strong>intent</strong>
+of such a <em>proof chain</em> is to model an approval chain in a business
+process or a notary witnessing analog signatures. In such cases we CANNOT allow
+the <em>proofs</em> that are relied upon to be <strong>substituted</strong> with
+other proofs or removed.
+      </p>
+      <p>
+Let's see how a <em>proof chain</em> is constructed where the VP of Engineering
+signs off on the document first, based on the VP of engineering's signature and
+a review the CFO then signs off of the document, finally based on both the prior
+signatures and a review the CEO signs off on the document. Since others will be
+referring to the VP of engineering's signature we need to add a <em>id</em> to
+the proof options. First the VC of engineering signs the VC:
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown, this proof set will verify
+  &quot;proof&quot;: {&quot;id&quot;: &quot;urn:proof-0&quot;, &quot;verificationMethod&quot;: pubVPE,
+    &quot;proofValue&quot;: signature(privVPE, document)}
+}
+      </code></pre>
+      <p>
+Next, the CFO receives the document, verifies that the VP of engineering signed
+it and signs it based on a review and on the VP of Engineering signing. For this
+we need to set up the <em>proof chain</em> by indicating a dependency on the
+proof in the document just received, i.e.,
+<code>previousProof = &quot;urn:proof-0&quot;</code>. The
+processing algorithm has us &quot;bind&quot; the previous proof to the unsigned
+document. Denote the this proof by <em>proof0</em> and &quot;binding&quot; this
+information to the document by <code>document + proof0</code>:
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown
+  &quot;proof&quot;: [
+  {&quot;id&quot;: &quot;urn:proof-0&quot;, &quot;verificationMethod&quot;: pubVPE,
+    &quot;proofValue&quot;: signature(VP_Eng, document)},
+  {&quot;id&quot;: &quot;urn:proof-1&quot;, &quot;verificationMethod&quot;: pubCFO, &quot;previousProof&quot;: &quot;urn:proof-0&quot;,
+    &quot;proofValue&quot;: signature(privCFO, document + proof0)}]
+}
+      </code></pre>
+      <p>
+Now when the CEO validates the received the VC with the above <em>proof
+chain</em> they will check that the CFO signed based on the VP of Engineering
+signing it. First, they will check the proof/signature with
+<code>id = 0</code> against the public key of the VP of Engineering. Note that
```

Comment:
Please use backticks instead of CODE markup because 1) it's more succinct, 2) easier to review (lower cognitive burden), and 3) broadly supported by ReSpec.